Fixed parameters' units and updated adapter to overfetch to fill data gaps for Taiwan

majesticio opened the pull request:
To be merged and deployed after faulty new measurements and rollups have been deleted.

russbiggs commented:
Based on the data gaps i was seeing, i'm not clear how an offset deployment would help. What is the behavior we see that necessitates an additional deployment? Can the same be achieved with overfetching in the main deployment?

We would want to have a clear case for an additional deployment, outlining what the pattern is and why the gaps appear.

majesticio commented:
I have looked at it some more and I think we can try to overfetch instead of using an extra deployment as the measurements endpoint is queryable by the hour historically, I can update the adapter to fetch the last 3 or 4 hours of measurments. this should resolve the data gaps issue.

majesticio commented:
I have updated the adapter to overfetch by 3 hours to address the data gaps. I removed the deployment as we are now relying on the over fetching strategy
